back/engines/commercial/verification/app/services/verification/patches/permissions/permissions_custom_fields_service.rb

```diff
@@ -16,7 +16,7 @@ def fields_for_permission(permission, return_hidden: false)
 
         # Add any fields that are locked to verification method
         def add_verification_fields(permission, fields)
-          method = VerificationService.new.first_method_enabled_for_verified_actions
+          method = VerificationService.new.first_method_enabled
```

Comment:
This is the main change on the BE: before, we would only show which fields were locked if a platform has a verification method that is supported for 'Verified actions'. At the moment, none are, except the Fake SSO. However, in our 'old' way of doing verification through smart groups, we can also have locked fields. In the previous behavior, admins would never see any locked fields, because none of them would have a 'Verified action' supported verification method. Now they will see which fields are locked if they have any verification enabled (the expected behavior).

I know all of this is quite a mindfuck so let me know if you need more explanation

Reply:
Thanks for the explanation, I think it's more or less clear 👍